CMakeLists.txt

```diff
@@ -22,46 +30,74 @@ target_link_directories(firebase INTERFACE
   third_party/firebase-development/usr/libs/windows/deps/app
   third_party/firebase-development/usr/libs/windows/deps/app/external)
 
-add_library(FirebaseCore
+add_library(FirebaseCore SHARED
```

Comment:
Is this change intentional?

Reply:
Yes, this change was intentional (IIRC, we ended up making them dynamically linked in Arc as well).